spec/system/admin/index_spec.rb

```diff
@@ -0,0 +1,5 @@
+describe 'Super Admin Landing Page' do
+  let(:url) { '/admin' }
+
+  it_behaves_like 'a page restricted to super admins'
```

Comment:
This spec is intentionally bare-bones - it's basically my sanity check to verify that the login contexts are working.